chart/openfaas/templates/faas-lambda-rbac.yaml

```diff
@@ -0,0 +1,94 @@
+{{- $functionNs := default .Release.Namespace .Values.functionNamespace }}
+{{- if eq .Values.faaslambda.create true }}
```

Comment:
Hey there!

What about using `{{- if .Values.faaslambda.create }}`? It's easier to read and more consistend with the other parts of the char :>